c/new-io.c

```diff
@@ -424,8 +421,6 @@ ptr S_bytevector_read(ptr file, ptr bv, iptr start, iptr count, IBOOL gzflag) {
   INT fd = gzflag ? 0 : GET_FD(file);
   glzFile gzfile = gzflag ? GZXFILE_GZFILE(file) : NULL;
 
- /* file is not locked; do not reference after deactivating thread! */
```

Comment:
Please leave the comment here, because it still applies.

Reply:
Comment restored